gdk/src/lib.rs

```diff
@@ -3,7 +3,6 @@
 #![cfg_attr(feature = "dox", feature(doc_cfg))]
 #![allow(clippy::missing_safety_doc)]
 #![allow(clippy::upper_case_acronyms)]
-#![allow(deprecated)]
```

Comment:
Because of this we get many warnings like

```
use of deprecated static `auto::constants::STYLE_REGION_TAB`: Since 3.20
```

now. Which is of course correct but also... why would it warn about that inside the crate that defines these things? :) Somehow you need to keep the deprecated API alive, and that requires using it inside the crate.

What should we should do about that?

Reply:
That's annoying indeed, but only happens when generating docs it seems. Maybe a rustdoc issue/behaviour?

Reply:
Ok :)